### PR TITLE
added Footer

### PR DESCRIPTION
**Pull request by AvineshTripathi**
Created a global footer 

### Approach

In Docusaurus we can modify the the default/core footer.js using the swizzle. More info https://docusaurus.io/docs/migration/manual#footer
The PR tries to make the Footer device independent, though there are scope of improvement.
We have used module.css to import css here. Refer https://dev.to/myogeshchavan97/an-introduction-to-css-modules-in-react-2fkd to know the benefits of module.css file 

### Screenshots

Screenshot from 2022-05-06 00-57-16
In inspect mode 
Screenshot from 2022-05-06 00-58-52
In full screen mode

cc @verma-kunal

### CODE REVIEW

File: src/theme/Footer/index.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  linkedin: {
+    Svg: require('@site/static/img/linkedIn.svg').default
+  }
+}
+
+function Footer() {
+  return (
+    <section className={styles.footer}>
+      <div>
+        <div className={styles.title}>help us do more</div>
```
Comment (SuperAayush) [inline src/theme/Footer/index.js:50]: Heading should start with a capital letter.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by SuperAayush — CHANGES_REQUESTED** [review]
1. You didn't use the font family which is given in the Figma file, please import the font family.
2. The Social icons you see in the Figma file are different from yours, they are black icons on white background unlike yours.
3. Add comments to differentiate the HTML and data part and for other purposes as well.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by AvineshTripathi** [comment]
> 1. You didn't use the font family which is given in the Figma file, please import the font family.
> 2. The Social icons you see in the Figma file are different from yours, they are black icons on white background unlike yours.
> 3. Add comments to differentiate the HTML and data part and for other purposes as well.
> 
> __ [sema-logo](https://semasoftware.com/gh)  **Summary:** 🛠️ This code needs a fix

1. I ll import that 
2. I couldn't find those similar logo so i couldn't, if you can provide me with those will be better 
3. I ll do that too

**Comment by verma-kunal** [comment]
Thanks for the PR and the ping @AvineshTripathi !! Will check this out asap

**Comment by AvineshTripathi** [comment]
Hey @SuperAayush I can see the font is already working, is there any particular reason why we want to import one, if it is not working for you or anyone elese then only I think we should import it. Thoughts?

**Comment by SuperAayush** [comment]
> Hey @SuperAayush I can see the font is already working, is there any particular reason why we want to import one, if it is not working for you or anyone elese then only I think we should import it. Thoughts?
> 
> Also by adding comments what exactly do you mean can you elaborate

So in the above preview, I see that the font is not as that used in the Figma file that is Poppins.
Refer to this link for the icons -> https://www.figma.com/file/g6OP4ozeVM8HzRVQ8CMALL/Kubesimplify-Website-Mockups?node-id=2%3A2
The icons are in the black color I believe you have to change the color to white while using them.
About the comments, I think the folks who will be working on the footer component in the future should be guided in brief using comments on what is in the file and about all the imports as well as a little about using the data dynamically or we can have it the readme on consulting with others.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

**Comment by AvineshTripathi** [comment]
@SuperAayush I have added comments, I can still not find logos so not adding now and the fonts mentioned in the figma is the same that I have used and I beleive its working, you can try cloning my forked repo and can add the changes to the `footer` branch

**Review by SuperAayush — APPROVED** [review]
LGTM 👍🏻

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Efficient

File: src/theme/Footer/index.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      <hr className={styles.hr}></hr>
+      <div className={styles.end_first}>© 2022 Kubesimplify</div>
+      <logo.mail.Svg className={styles.mail_logo} row="img" />
+      <div className={styles.mail_title}>example@kubesimplify.com</div>
+      <div className={styles.group_logo}>
+        <logo.twitter.Svg row="img" />
+        <logo.instagram.Svg row="img" />
+        <logo.github.Svg row="img" />
+        <logo.linkedin.Svg row="img" />
+        <logo.youtube.Svg row="img" />
```
Comment (SuperAayush) [inline src/theme/Footer/index.js:79]: Add redirect links with the target as blank to the logos to different kubesimplify socials

**Review by SuperAayush — CHANGES_REQUESTED** [review]

**Comment by AvineshTripathi** [comment]
I have taken the logos from the figma desgin and placed it, but for me it showed a bug where all the logo in footer rendered same(whichever we mention first). I tried solving but didn't find any solution anyone who is willing to take this up and fix this bug that would be helpful. 

@anubha-v-ardhan do we need to wait to get this bug fixed as I am not finding any problem with the codebase mostly this is a bug which can be fixed with another PR so can we merge this and somebody can do a PR solving this bug 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by verma-kunal** [comment]
> I have taken the logos from the figma desgin and placed it, but for me it showed a bug where all the logo in footer rendered same(whichever we mention first). I tried solving but didn't find any solution anyone who is willing to take this up and fix this bug that would be helpful.
> 

Hey @AvineshTripathi , were you able to solve this issue?

**Comment by AvineshTripathi** [comment]
Still couldn't get a solution to this I tried various stuff but still the bug remains the same

File: src/theme/Footer/index.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  sponsorsContent: (
+    <>
+      If you like our work, you can <b>sponsor & support</b> the community!<br></br>
+      All the money will be used to continue our mission of <b>providing simplified cloud native education to all</b> and make this community reach more number of people.
+    </>
+  )
+}
+
+const logo = {
+  hashnode: {
```
Comment (Shivansh-yadav13) [inline src/theme/Footer/index.js:24]: @AvineshTripathi what are your thoughts on using something like react-icons?

Comment (AvineshTripathi) [reply src/theme/Footer/index.js:24]: Any particular reason going for react-icons?

**Comment by AvineshTripathi** [comment]
@verma-kunal can we merge this and I ll create another issue for the bug and folks here can solve it

**Comment by verma-kunal** [comment]
> @verma-kunal can we merge this and I ll create another issue for the bug and folks here can solve it 

Sounds good @AvineshTripathi !! Was just waiting for your confirmation on the issues on this PR

**Comment by verma-kunal** [comment]
Thanks for contributing to this @AvineshTripathi ✨

**Pull request merged**